README.md

```diff
@@ -136,23 +136,23 @@ Yes, until the resource is loaded the html will be seen unstyled. You could get
 
 ### Will this make my site faster?
-Hopefully Yes! If you are suffereing from css bloat ie. you have a 500kb css file that is loaded on every page. It would make sense to componentize it, and load it only when needed. This will cut down your file size and it should also make maintaining it easier.
+Hopefully, yes! If you are suffereing from css bloat (_i.e._, you have a 500kb css file that is loaded on every page), it would make sense to componentize it, and load it only when needed. This will cut down your file size and it should also make maintaining it easier.
 
-Also if you have some html which is the same on every page (like the header, footer and sidebar) Then loading them as components will force the browser cache them, so when a user browsers your site these html resources will be loaded instantly and the html pages will be smaller.
+Also, if you have some html which is the same on every page (like the header, footer and sidebar), then loading them as components will force the browser cache them. So, when a user browses your site these html resources will be loaded instantly and the html pages will be smaller.
 
 ### What about content that is added to the DOM dynamically?
-The `carl.min.js` is loaded once just before the closing body tag, so any css selectors or window objects would have to be already present in the DOM. The script can of cource be called again as a callback after you've loaded your dynamic content, then all the tests would be ran again.
+The `carl.min.js` is loaded once just before the closing body tag, so any css selectors or window objects would have to be already present in the DOM. The script can of cource be called again as a callback after you've loaded your dynamic content, then all the tests would be run again.
 
 ### Can I include scripts in html resources?
-Yes. Any script that is included in a html resource (like the footer.html) will be executed
+Yes. Any script that is included in a html resource (like the `footer.html`) will be executed.
 
-### How can I gaurantee that a script will fire after a html resource has been loaded
-You would need to include the script inside the html resource to absolutely gaurentee that it is loaded after the html has been added to the DOM.
+### How can I guarantee that a script will fire after an html resource has been loaded?
+You would need to include the script inside the html resource to absolutely guarantee that it is loaded after the html has been added to the DOM.
 
-Ajax is Asyncronous (its what the "A" stands for). Because of this there is no gaurantee that one resource is loaded before the other, so rather than doing this...
+Ajax is Asyncronous (its what the "A" stands for). Because of this there is no guarantee that one resource is loaded before the other. So, rather than doing this...
```

Comment:
```suggestion
Ajax is Asyncronous (it's what the "A" stands for). Because of this there is no guarantee that one resource is loaded before the other. So, rather than doing this...
```